srcDoc/release_notes.rst

```diff
@@ -48,6 +48,19 @@ releases:
 Release Notes Master
 --------------------
 
+What's New in Master
```

Comment:
Hi @javmorcas , can you take a look at the release note of this feature. 
Thank you